File: libbase/src/main/java/com/aquila/lib/base/OnViewClickListener.java
package com.aquila.lib.base;

import android.view.View;

/***
 * @date 创建时间 2020/3/23 10:22
 * @author 作者: W.YuLong
 * @description
 */
public interface OnViewClickListener {
    /* 通用的点击事件接管*/
    <T> void onClickAction(View v, String str, T t);
}

File: libbase/src/main/java/com/aquila/lib/base/OnViewClickListener.kt
package com.aquila.lib.base

import android.view.View

/***
 *@date 创建时间 2019-09-26 13:05
 *@author 作者: W.YuLong
 *@description  点击事件的接口
 */
interface OnViewClickListener {

    /* 通用的点击事件接管*/
    fun <T> onClickAction(v: View, tag: String, t: T?)
}
